modules/installation-dns-user-infra.adoc

```diff
@@ -44,15 +44,15 @@ API calls can fail, and you cannot retrieve logs from Pods.
 
 |Routes
 |`*.apps.<cluster_name>.<base_domain>`
-|A wildcard DNS record that points to the load balancer that targets the
-machines that  run the Ingress router pods, which are the worker nodes by
+|A wildcard DNS A/AAAA or CNAME record that points to the load balancer that targets the
+machines that run the Ingress router pods, which are the worker nodes by
 default. This record must be resolvable by both clients external to the cluster
 and from all the nodes within the cluster.
 
 .2+|etcd
 |`etcd-<index>.<cluster_name>.<base_domain>`
-|{product-title} requires DNS records for each etcd instance to point to the
-control plane machines that host the instances. The etcd instances are
+|{product-title} requires DNS A/AAAA records for each etcd instance to point to the
+control plane machines that host the instances.  The etcd instances are
```

Comment:
There's an extra space here. Otherwise, this LGTM!

Reply:
Thanks for the review.  Fixed the space